app/mbm/templates/mbm/index.html

```diff
@@ -9,6 +9,24 @@
   <link rel="stylesheet" href="{% static 'css/spin.css' %}" />
 {% endblock %}
 
+{% block head_js %}
```

Comment:
**[Nitpick, non-blocking]** I wonder if it would make more sense to include this in `base.html` instead of an override in `index.html`, so we can track visits to other pages? Currently the only other page is `About` I think, so it's really not a big deal either way 🤷🏻‍♀️

Reply:
I forgot about the About page actually. I was meaning to exclude it from the admin pages but there's no real need. I'll put it in the base template.